server/internal/app/i18n.go

```diff
@@ -0,0 +1,57 @@
+package app
+
+import (
+	"embed"
+
+	"github.com/labstack/echo/v4"
+	"github.com/nicksnyder/go-i18n/v2/i18n"
+	"github.com/reearth/reearth-cms/server/internal/adapter"
+	"github.com/reearth/reearthx/log"
+	"golang.org/x/text/language"
+	"gopkg.in/yaml.v3"
+)
+
+var (
+	//go:embed i18n/locale.*.yaml
```

Comment:
```suggestion
	//go:embed ../../i18n/locale.*.yaml
```

Reply:
`go:embed` does not support parent dir you can check this issue https://github.com/golang/go/issues/46056

Reply:
I got